pegasus/helpers/page_helpers.rb

```diff
@@ -70,18 +70,3 @@ def combine_css(*paths)
   ).render
   [css_min, css_last_modified]
 end
-
-# Returns a random donor's twitter handle.
-def get_random_donor_twitter
```

Comment:
was this used on other pages that might have a problem now? it's hard for me to tell what imports what in ruby.

or does it not matter since you added the new file to application_helper?

Reply:
Added back the import for Pegasus in commit https://github.com/code-dot-org/code-dot-org/commit/a5479eccff3cef85aa2ae5894e5e7bb62f49d559.